kitsune/community/utils.py

```diff
@@ -18,17 +21,23 @@
 def top_contributors_questions(start=None, end=None, locale=None, product=None, count=10, page=1):
     """Get the top Support Forum contributors."""
 
+    if not start:
+        start = datetime.now() - timedelta(days=90)
+
+    limit = count * 10
```

Comment:
Shall we cap the count to number that makes sense. The code is smart and efficient but it can become a problem if the function is called with a count of 10K for example